Sources/Core/AWSSDKIdentity/Sources/AWSSDKIdentity/AWSCredentialIdentityResolvers/SSOAWSCredentialIdentityResolver.swift

```diff
@@ -58,15 +58,6 @@ public struct SSOAWSCredentialIdentityResolver: AWSCredentialIdentityResolver {
     }
 
     public func getIdentity(identityProperties: Attributes?) async throws -> AWSCredentialIdentity {
-        guard let identityProperties, let internalSSOClient = identityProperties.get(
-            key: InternalClientKeys.internalSSOClientKey
-        ) else {
-            throw AWSCredentialIdentityResolverError.failedToResolveAWSCredentials(
-                "SSOAWSCredentialIdentityResolver: "
-                + "Missing IdentityProvidingSSOClient in identity properties."
-            )
-        }
-
```

Comment:
No need to get the internal SSO client from identity properties anymore, since it can now be directly created by this type (just below).

Reply:
Same change made in the following identity resolvers.